rfd/0180-sso-as-mfa-method.md

```diff
@@ -72,171 +194,197 @@ If browser window does not open automatically, open it by clicking on the link:
  http://127.0.0.1:60433/f5858c78-75e1-4f3f-b2c5-69e8e76c0ff9
 ```
 
-If SSO is not the required MFA method in the cluster, the user will also be
-notified of how to add an MFA device for future MFA checks. The SSO login
-browser will not be opened automatically in order to draw attention to the
-output. The link can still be clicked for easy UX.
-
-```console
-> tsh ssh server01
-MFA is required to access Node "server01"
-Complete an SSO auth check in your local web browser. Open it by clicking on the link:
- http://127.0.0.1:60433/f5858c78-75e1-4f3f-b2c5-69e8e76c0ff9
-Or register an MFA device with `tsh mfa add` to complete MFA checks with a tap.
-```
-
-Note: the MFA prompt is moved to the end to help it stick out over the SSO link.
-
-Stretch: Rather than having the user add an MFA device with additional steps,
-they should be guided through MFA registration and allowed to complete their
-request afterwards. This would also apply without the SSO MFA option.
+##### Adding first MFA Device
 
-```console
-> tsh ssh server01
-MFA is required to access Node "server01"
-Complete an SSO auth check in your local web browser. Open it by clicking on the link:
- http://127.0.0.1:60433/f5858c78-75e1-4f3f-b2c5-69e8e76c0ff9
-Or register an MFA device to complete MFA checks with a tap.
-Choose device type [WEBAUTHN]:
+Usually, a new SSO user without a registered MFA device can add their first
+device without an additional authentication check. With SSO MFA, we can include
+an MFA check even for the first device.
```

Comment:
Does this mean the check would be required, as it normally would if the user had another device?

I guess what I'm asking is if the backend considers the SSO MFA device when counting the user devices.